### PR TITLE
Deploy tags automatically

### PR DESCRIPTION
**Pull request by anxolin**
**Description**
Prepare PRODUCTION deploy automatically on every tag.

**Why**
Because before we needed to tag, and then create a new release. We want to prepare for deployment every tag.

Also, there were some small errors explained in the details

**Details**
Now it gives better output. It prints the current version (so we also make sure the env var is correctly set)

Also, it will inform about the passed parameters. If they are not available:
image

Additionally, now returns an error in that case. Before this PR it was just printing the error, and showing success (ends with a 0 error code). So now, if any param is missing, now it returns the `100` error code.

The error message, now is printed in stderr, instead of stdout (more correct, but also GA understand it and display it better)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@gnosis.pm/gp-v1",
-  "version": "2.1.0-rc.0",
+  "version": "2.1.0-rc.17",
```
Comment (alfetopito) [inline package.json:3]: 17? LoL

Comment (anxolin) [reply package.json:3]: Yes!

I was commenting most of the cy.yml and making looots of tests :)

**Review by davidalbela — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -153,10 +152,13 @@ jobs:
...
   deploy:
     name: Deploy
-    needs: [build, storybook]
+    # needs: [build, storybook]
     runs-on: ubuntu-latest
 
     steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
```
Comment (alfetopito) [inline .github/workflows/ci.yml:161]: Don't care about the storybook, but no longer depends on `build`?

I see you added the code checkout step, but why? Need to have the source in order to push stuff?

Comment (anxolin) [reply .github/workflows/ci.yml:161]: yes, that was a test!
forgot to uncomment

**Pull request merged**